doc/glossary.rst

```diff
@@ -169,6 +169,16 @@ general neuroimaging concepts. If you think a term is missing, please consider
         it is a one-dimensional time series capturing the spatial variability
         of the signal across sensor locations.
 
+    HED
+    hierarchical event descriptors
+        Hierarchical event descriptors (abbr. ``HED``) are tags that can be
+        used to describe many different types of EEG experiment events (for 
+        example, stimulus/circle/red/left and stimulus/square/blue/left).
+        These are organized hierarchically such that the first labels are fixed
+        and corresponding descriptors can be extended to describe any event
+        with any level of detail. HED tags make it easy to search through and
+        analyze variations of events of the same type.
```

Comment:
@hoechenberger I am uneasy on this. HED from the eeglab folks is more than the tagging system we use with the `/`

At beast we support a subset of HED I think

Reply:
I **thought** the forward slashes were pretty much what made up the first revision of HED. I need to have a look at the paper again, but I believe they didn't even specify pre-defined terms – this only came in later revisions of HED

Reply:
> At best we support a subset of HED I think

I agree, this is what I raised in https://github.com/mne-tools/mne-python/issues/8974#issuecomment-790737767

We should read the HED (1st generation) paper (so the very first HED paper) carefully to check, and adjust the proposed docs here to include this as a caveat.

Importantly, HED is a "standard" that has evolved in the past (and continuous to evolve). At its current stage it goes way way beyond what we do in MNE.